src/services/authentication.js

```diff
@@ -1683,6 +1689,40 @@ function getSSOLoginUrlClient ({ client, authorizationParams, ssoConnection }) {
   return ssoLoginUrl
 }
 
+function getTokenTypeTransformationsOrder (provider) {
+  if (provider === 'facebook') {
+    return [
+      'CAPITALIZE',
```

Comment:
You could add a comment about why you need to capitalize facebook token type unlike other providers.